audiomentations/augmentations/equalizer.py

```diff
@@ -0,0 +1,94 @@
+from audiomentations import LowShelfFilter, PeakingFilter, HighShelfFilter
+from audiomentations.core.transforms_interface import BaseWaveformTransform
+
+
+class Equalizer(BaseWaveformTransform):
+    """
+    Adjust the volume of different frequency bands. This transform is a 7-band
+    parametric equalizer - a combination of one low shelf filter, five peaking filters
+    and one high shelf filter, all with randomized gains, q values and center frequencies.
+    """
+
+    supports_multichannel = True
+
+    def __init__(
+        self,
+        min_gain_in_db: float = -12.0,
+        max_gain_in_db: float = 12.0,
+        p: float = 0.5,
+    ):
+        """
+        :param p: The probability of applying this transform
+        """
+        super().__init__(p)
+        assert min_gain_in_db <= max_gain_in_db
+
+        self.low_shelf_filter = LowShelfFilter(
```

Comment:
How were those options chosen?

Reply:
I used the same center freq means as the defaults in Fruity Parametric EQ 2. The min-max values were used to give them some wiggle room (a "perfect fifth" up or down) without having too much overlap potential between the filters